OmiseSDKTests/Fixtures/objects/capability.json

```diff
@@ -186,6 +186,93 @@
               "active": true
           }
       ]
+    },
+    {
+      "object": "payment_method",
+      "name": "alipay_cn",
+      "currencies": [
```

Comment:
I'm worried hardcoding this feels a bit hacky. Do we need to update this everytime our processor changes their support? If this is the only way to do it then that's fine but just wondering.

Reply:
I believe this is only a fixture for the test, the actually payment supported currencies should be fetched from the capabilities api

Reply:
yep you're right, I missed that it was a test fixture 🤦🏽